publicCloudInfo-server.conf.template

```diff
@@ -1,13 +1,13 @@
 <VirtualHost *:80>
   ServerAdmin jgordon@suse.com
```

Comment:
We might want to remove @joelgordon's email from here :slightly_smiling_face:

Reply:
It can be changed to publiccloud-admin@susecloud.net

Reply:
Which email should we use here? Or should we just remove the ServerAdmin config?

Reply:
@guangyee well, it's a template, so I'd go for some dummy address like `email@example.org`.